tests/php/test-fieldmanager.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Tests for functions in fieldmanager.php.
+ */
+
+class Test_Fieldmanager extends WP_UnitTestCase {
+	public function test_fm_failed_validation_to_wp_die() {
+		$rand_str = rand_str();
+
+		try {
+			fm_failed_validation_to_wp_die( $rand_str );
```

Comment:
If this doesn't `wp_die()`, this test will pass because the exception will not be thrown. I think the best bet is to set `@expectedException WPDieException` and after your assertion, `throw $e;`